### PR TITLE
Changes according to suggestions

### PR DESCRIPTION
**Pull request by Koushith**
### Changes asked for - 

- Add a floating action button. 
- Add a card with dismissing, 
- card with text overlay 
- Add separate examples of each card Variant, refer cards from LMS. 
- Add Examples of typography in the documentation.

### Changes - 

- [x] -Add a floating action button. 

image

- [x] Add a card with dismissing, 
image

- [x]  card with text overlay 
image

- [x] Add separate examples of each card Variant, refer cards from LMS. 
- [x] Add Examples of typography in the documentation.

### CODE REVIEW

File: src/components/card/card.component.html [hunk 1]
```diff
@@ -246,6 +248,80 @@ <h2 class="card-title">Mi Band</h2>
...
+              <div class="close">x</div>
+              <div class="action-container">
+                <h2 class="card-title">Mi Band</h2>
+                <div class="card-text">Smartwatches 2nd Gen</div>
+                <div class="card-price">
+                  <span class="price-now"> Rs.490 </span>
+                  <span class="discount"> (62% OFF) </span>
+                </div>
+                <span class="overlay-text">Out of Stock</span>
+                <button class="btn btn-primary">ADD TO CART</button>
```
Comment (ajit-sharmaGH) [inline src/components/card/card.component.html:312]: Good naming convention

**Review by ajit-sharmaGH — APPROVED** [review]
You did awesome 
Code looks clean
used good way naming
You can merge it

File: index.css [hunk 2]
```diff
@@ -104,7 +105,7 @@ h6 {
 
 .dark {
   background-color: black;
-  color: white;
+  color: var(--neutral-white);
 }
 
 .centered {
```
Comment (dikshadhote) [inline index.css:111]: use proper naming convention

File: src/components/button/buttons.component.css [hunk 3]
```diff
@@ -14,9 +14,7 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  background: var(--primary-dark);
-  color: var(--neutral-white);
-  font-size: 18px;
+  font-size: 1.8rem;
   font-weight: 500;
   min-width: 100px;
```
Comment (dikshadhote) [inline src/components/button/buttons.component.css:19]: use rem for values greater than 10px

File: src/components/card/card.component.css [hunk 4]
```diff
@@ -146,3 +177,31 @@
...
+
+/* ---------CARD WITH OVERLAY ------- */
+.ecom-overlay {
+  width: 23rem;
+  box-shadow: none;
+  cursor: pointer;
+  transition: all 0.2s ease-in-out;
+  position: relative;
+  padding-bottom: 1.4rem;
+  background-color: #f9f7fb;
```
Comment (dikshadhote) [inline src/components/card/card.component.css:189]: use variable name for color

**Review by dikshadhote — CHANGES_REQUESTED** [review]
Please structure your code properly.
1.Code snippet for display card, floating action button is missing 
image
2. Typography is reflecting twice 
image
3. Use proper alignment for code snippets
image

File: src/components/card/card.component.css [hunk 5]
```diff
@@ -59,20 +60,20 @@
   background-color: var(--primary-main);
   color: white;
```
Comment (Prathyusharam) [inline src/components/card/card.component.css:61]: You can make use of a variable here.

**Review by Prathyusharam — COMMENTED** [review]
great work 
UI looks clean and good
good to go

**Review by Prathyusharam — APPROVED** [review]

File: src/components/card/card.component.css (hunk 4 above)
Comment (Koushith) [reply src/components/card/card.component.css:189]: updated it in the latest commit.

**Pull request merged**